nixos/modules/services/networking/networkmanager.nix

```diff
@@ -369,16 +369,38 @@ in {
         '';
       };
 
-      enableFccUnlock = mkOption {
+      enableBundledFccUnlockScripts = mkOption {
         type = types.bool;
         default = false;
         description = lib.mdDoc ''
-          Enable FCC unlock procedures. Since release 1.18.4, the ModemManager daemon no longer
+          Enable FCC unlock procedures shipped with ModemManager.
+          Since release 1.18.4, the ModemManager daemon no longer
           automatically performs the FCC unlock procedure by default. See
           [the docs](https://modemmanager.org/docs/modemmanager/fcc-unlock/)
```

Comment:
```suggestion
          [the modemmanager docs](https://modemmanager.org/docs/modemmanager/fcc-unlock/)
```

Reply:
This is unrelated to this PR and was already present before.